OpenSim/Actuators/MuscleFirstOrderActivationDynamicModel.cpp

```diff
@@ -160,14 +79,38 @@ calcDerivative(double activation, double excitation) const
 {
```

Comment:
That's for when this component has its own state variable, right? I don't think Tom is tackling that in this pR.